### DISCUSSION TITLE
package.json에서 사용되지 않는 라이브러리 제거하시는 거 어떻게 생각하시나요?

### DISCUSSION THREAD

**Discussion by judahhh** [coffee-meet/frontend#243] [잡담]
설치한 후에 잘 사용되지 않는 라이브러리에 대해서 필요성을 논의하고 정리하는게 필요할 것 같아요!

**Comment by changgihong** [comment]
저는 d3가 일단 1순위일 것 같아요..!
프로젝트 내에서 타이머를 구현하는 데에만 사용하고 있는데, 타이머 자체는 d3의 타이머를 이용하지 않고도 충분히 구현할 수 있을 뿐더러, d3는 본래 데이터 시각화 라이브러리인만큼 용도에 맞게 사용하는게 좋다고 생각합니다!
